napari/layers/_base_layer/view/properties.py

```diff
@@ -147,3 +162,8 @@ def _on_blending_change(self, event):
     def _on_visible_change(self, event):
         with self.layer.events.visible.blocker():
             self.visibleCheckBox.setChecked(self.layer.visible)
+
+    def _on_thumbnail_change(self, event):
+        image = QImage(self.layer.thumbnail, self.layer.thumbnail.shape[1],
```

Comment:
this is repeated – may want to look into putting this logic into its own function and then connect it into an event, wrapping it with a lambda

Reply:
removed the extra function call